vendor/osxlockdown/commands.yaml

```diff
@@ -0,0 +1,276 @@
+  fix_command: |
+    cupsctl --no-share-printers
+  enabled: true
+
+
+- title: "Disable Wake on Network Access"
+  check_command: |
+    sudo systemsetup getwakeonnetworkaccess | grep "Wake On Network Access: Off"
+  fix_command: |
+    sudo systemsetup -setwakeonnetworkaccess off
+  enabled: true
+
+
+- title: "Disable File Sharing"
+  check_command: |
+    if [ -n "$(launchctl list | egrep AppleFileServer)" ]; then exit 1; fi; if [ -n "$(grep -i array /Library/Preferences/SystemConfiguration/com.apple.smb.server.plist)" ]; then exit 1; fi; exit 0;
+  fix_command: |
+    launchctl unload -w /System/Library/LaunchDaemons/com.apple.AppleFileServer.plist; launchctl unload -w /System/Library/LaunchDaemons/com.apple.smbd.plist
+  enabled: true
+
+
+- title: "Disable Remote Management"
+  check_command: |
+    if [ -n "$(ps -ef | egrep "/System/Library/CoreServices/RemoteManagement/ARDAgent.app/Contents/MacOS/[A]RDAgent")" ]; then exit 1; fi; exit 0
+  fix_command: |
+    sudo /System/Library/CoreServices/RemoteManagement/ARDAgent.app/Contents/Resources/kickstart -deactivate -stop
+  enabled: true
+
+
+- title: "Enable FileVault"
+  check_command: |
+    diskutil cs list | grep -i "Encryption Status:       Unlocked"
+  comment: "TODO: FileVault can be enabled with `fdesetup enable`, but you need to enter user creds and write down the recovery key. How to do this programmatically, or at least tell the user the steps?  See https://derflounder.wordpress.com/2013/10/22/managing-mavericks-filevault-2-with-fdesetup/"
+  enabled: true
+
+
+- title: "Destroy File Vault Key when going to standby"
+  check_command: |
+    pmset -g | grep DestroyFVKeyOnStandby | grep 1
+  fix_command: |
+    sudo pmset -a destroyfvkeyonstandby 1
+  enabled: true
+
+
+- title: "Enable hibernation mode (no memory power on sleep)"
+  check_command: |
+    pmset -g | grep hibernatemode | grep 25
+  fix_command: |
+    sudo pmset -a hibernatemode 25
+  enabled: true
+
+
+- title: "Enable Gatekeeper"
+  check_command: |
+    spctl --status | grep "assessments enabled"
+  fix_command: |
+    sudo spctl --master-enable
+  enabled: true
+
+
+- title: "Enable Firewall"
+  check_command: |
+    test $(defaults read /Library/Preferences/com.apple.alf globalstate) -ge 1
+  fix_command: |
+    defaults write /Library/Preferences/com.apple.alf globalstate -int 1
+  enabled: true
+
+
+- title: "Enable Firewall Stealth Mode"
+  check_command: |
+    /usr/libexec/ApplicationFirewall/socketfilterfw --getstealthmode | grep "Stealth mode enabled"
+  fix_command: |
+    /usr/libexec/ApplicationFirewall/socketfilterfw --setstealthmode on
+  enabled: true
+
+
+- title: "Disable signed apps from being auto-permitted to listen through firewall"
+  check_command: |
+    defaults read /Library/Preferences/com.apple.alf allowsignedenabled | grep 0
+  fix_command: |
+    defaults write /Library/Preferences/com.apple.alf allowsignedenabled -bool false
+  enabled: true
+
+
+- title: "Disable iCloud drive"
```

Comment:
Almost every Mac users store some stuff in iCloud, I don't see any any reason why we should disable it